Bottom: dccfd9e887b60ef0817741d3115044a0e65b10f4
Top:    2283372d1a87396c186806d58db7a2ac1925c871
Author: Jason Wang <jasowang@redhat.com>
Date:   2017-03-20 14:01:31 +0800

Refresh of tun-introduce-0

---

diff --git a/include/linux/if_tun.h b/include/linux/if_tun.h
index 40a39d8..149fc0d 100644
--- a/include/linux/if_tun.h
+++ b/include/linux/if_tun.h
@@ -19,7 +19,7 @@
 
 #if defined(CONFIG_TUN) || defined(CONFIG_TUN_MODULE)
 struct socket *tun_get_socket(struct file *);
-struct skb_array *tap_get_skb_array(struct file *);
+struct skb_array *tun_get_skb_array(struct file *);
 #else
 #include <linux/err.h>
 #include <linux/errno.h>
@@ -29,7 +29,7 @@ static inline struct socket *tun_get_socket(struct file *f)
 {
 	return ERR_PTR(-EINVAL);
 }
-static inline struct socket *tap_get_skb_array(struct file *f)
+static inline struct socket *tun_get_skb_array(struct file *f)
 {
 	return ERR_PTR(-EINVAL);
 }
